obip-ts-reports.md

```diff
@@ -0,0 +1,71 @@
+__OBIP__: TS-Reports
+
+__Title__: 3rd Party Search Content reporting
+
+__Author__: Tyler Smith <tyler@ob1.io>
+
+__Discussions-To__: Github Issues, #3rd-party-search on Slack, or <tyler@ob1.io>  
+
+__Status__: Draft
+
+__Type__: Standards Track
+
+__Created__: 10/03/2017
+
+__Copyright__: MIT
+
+
+## Abstract
+A specification for the API a search provider should comply with to handle client reports for content.
+
+
+## Motivation
+Search providers have variying content policies and need a way to get reports of infringing content. This Proposal suggests an API for managing these reports.
```

Comment:
Typo - "varying"

Reply:
Thanks, I will fix this. Speling isn't my stong suit.